docs/t8ntool-ref.rst

```diff
@@ -0,0 +1,298 @@
+  * `status`
+  * `cumulativeGasUsed`
+  * `logsBloom`
+  * `logs`
+  * `transactionHash`
+  * `contractAddress`, the address of the created contract, if any
+  * `gasUsed`
+  * `blockHash`, all zeros because this is created before the block is finished
+  * `transactionIndex`
+
+
+Example
+^^^^^^^
+
+::
+
+   {
+     "stateRoot": "0x1c99b01120e7a2fa1301b3505f20100e72362e5ac3f96854420e56ba8984d716",
+     "txRoot": "0xb5eee60b45801179cbde3781b9a5dee9b3111554618c9cda3d6f7e351fd41e0b",
+     "receiptRoot": "0x86ceb80cb6bef8fe4ac0f1c99409f67cb2554c4432f374e399b94884eb3e6562",
+     "logsHash": "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
+     "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+     "receipts": [
+        {
+            "root": "0x",
+            "status": "0x1",
+            "cumulativeGasUsed": "0xa878",
+            "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+            "logs": null,
+            "transactionHash": "0x4e6549e2276d1bc256b2a56ead2d9705a51a8bf54e3775fbd2e98c91fb0e4494",
+            "contractAddress": "0x0000000000000000000000000000000000000000",
+            "gasUsed": "0xa878",
+            "blockHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+            "transactionIndex": "0x0"
+        }
+     ]
+   }
+
+
+
+Trace Files
+-----------
+If **-\\-trace** is specified, the t8ntool creates a file (or files) called 
+`trace-<transaction number>-<transaction hash>.jsonl`. The format of this file
+is specified in 
+`EIP 3155 <https://github.com/ethereum/EIPs/blob/master/EIPS/eip-3155.md>`_.
+
+If the transaction fails and does not produce a hash, the name of the file is
+still `trace-<transaction number>-<value that is a legitimate hash>.jsonl`.
+
+
+
+Using Standard Input and Output
+===============================
+It should also be possible to run a `t8ntool` with the input coming from `stdin`
+and the output going to `stdout`. In this case, the input is all one object and
+the output is all one object.
```

Comment:
Clarification: 
The input/output is all one object, yes, but you can still have `txs` and `alloc` coming as `stdin` and `env` as file, and send `result` to file but send `alloc` to `stdout`.